src/ocm/hooks/useClusterPreflightRequirements.ts

```diff
@@ -3,21 +3,24 @@ import { alertsSlice } from '../../common/reducers';
 import { Cluster, PreflightHardwareRequirements } from '../../common/api/types';
 import { ClustersAPI } from '../services/apis';
 import { getErrorMessage, handleApiError } from '../api';
+import { APIErrorMixin } from '../api/types';
+
+type ErrorType = APIErrorMixin & Error;
 
 const { addAlert } = alertsSlice.actions;
 
 const useClusterPreflightRequirements = (clusterId: Cluster['id']) => {
   const [preflightRequirements, setPreflightRequirements] =
     React.useState<PreflightHardwareRequirements>();
-  const [error, setError] = React.useState();
+  const [error, setError] = React.useState<ErrorType>();
 
   React.useEffect(() => {
     const fetchFunc = async () => {
       try {
         const { data } = await ClustersAPI.getPreflightRequirements(clusterId);
         setPreflightRequirements(data);
       } catch (e) {
-        setError(e);
+        setError(e as ErrorType);
```

Comment:
This is how I fix it in my PR https://github.com/openshift-assisted/assisted-ui-lib/pull/1458/files#diff-fb2591b635c55d45a38368c2a889bbb2558786a5aa870ad0cfbcd47b2b7451cfR21

Reply:
I'll revert this change in the PR given that you have addressed it in a better way in yours.